### ISSUE TITLE
Misleading project name (should be mqtt2homekit)

### ISSUE THREAD

**Issue by fredck** [hobbyquaker/homekit2mqtt#170]
The scope of this project is to expose MQTT devices into HomeKit. Therefore, I believe the correct name for it should be **mqtt2homekit**.

Please correct me if I'm wrong.

---

I reached this repository because I was searching for the opposite: to expose HomeKit devices into MQTT (hence the name confusion).

The problem I'm trying to solve is controlling some HomeKit devices which cannot be added to Home Assistant (e.g. Thread enabled devices like Nanoleak and Eve stuff) with Home Assistant through MQTT.

**Comment by ultrathew** [comment]
Hi @fredck I'm in the exact same situation. Let me know if you find a solution, and I'll do the same.

**Comment by noelhibbard** [comment]
Agreed on the poor naming. I am also looking for a way to programmatically control native HomeKit devices and thought this was the solution. The search continues.

**Comment by benjiwolff** [comment]
If what you're saying is true, the main README is misleading as well. It says "HomeKit to MQTT bridge 🏡📱" without specifying that it is only in the direction from MQTT to HomeKit.
